inc/uninstall.class.php

```diff
@@ -446,51 +447,57 @@ public static function deleteComputerInOCS($ocs_id, $ocs_server_id)
     {
         /** @var DBmysql $DB */
         global $DB;
-        //@phpstan-ignore-next-line
-        $DBocs = PluginOcsinventoryngOcsServer::getDBocs($ocs_server_id)->getDB();
-
-        //First try to remove all the network ports
-        $query = "DELETE
-                FROM `netmap`
-                WHERE `MAC` IN (SELECT `MACADDR`
-                                FROM `networks`
-                                WHERE `networks`.`HARDWARE_ID` = '" . $ocs_id . "')";
-        $DBocs->query($query);
-
-        $tables =  ["accesslog", "accountinfo", "bios", "controllers", "devices", "drives",
-            "download_history", "download_servers", "groups_cache", "inputs",
-            "memories", "modems", "monitors", "networks", "ports", "printers",
-            "registry", "slots", "softwares", "sounds", "storages", "videos"
-        ];
+        if (class_exists('PluginOcsinventoryngOcsServer')) {
+            $DBocs = PluginOcsinventoryngOcsServer::getDBocs($ocs_server_id)->getDB();
+
+            //First try to remove all the network ports
+            $query = "DELETE
+                    FROM `netmap`
+                    WHERE `MAC` IN (SELECT `MACADDR`
+                                    FROM `networks`
+                                    WHERE `networks`.`HARDWARE_ID` = '" . $ocs_id . "')";
+            $DBocs->query($query);
+
+            $tables =  ["accesslog", "accountinfo", "bios", "controllers", "devices", "drives",
+                "download_history", "download_servers", "groups_cache", "inputs",
+                "memories", "modems", "monitors", "networks", "ports", "printers",
+                "registry", "slots", "softwares", "sounds", "storages", "videos"
+            ];
 
-        foreach ($tables as $table) {
-            if (self::ocsTableExists($ocs_server_id, $table)) {
-                $query = "DELETE
-                      FROM `" . $table . "`
-                      WHERE `hardware_id` = '" . $ocs_id . "'";
-                $DBocs->query($query);
+            foreach ($tables as $table) {
+                if (self::ocsTableExists($ocs_server_id, $table)) {
+                    $query = "DELETE
```

Comment:
This should have been replaced with a properly built query, no a `doQuery` call.